src/app/services/api.service.ts

```diff
@@ -0,0 +1,22 @@
+import { HttpClient } from "@angular/common/http";
+import { Observable } from "rxjs";
+
+export class ApiService {
+    baseURL: string = "SomeURL" // Idk if this is where we wanna have this?
```

Comment:
Can we please make use of angular environments
### ng g environment

Reply:
yes